src/pages/auth/auth.html

```diff
@@ -13,11 +13,11 @@
         <ion-col>
                 <img src="./assets/imgs/oidc.png" class="openid-logo"/>
```

Comment:
@StephenCoady perhaps remove the unnecessary indentation here as well. Otherwise LGTM